src/libraries/Microsoft.Bcl.Numerics/ref/Microsoft.Bcl.Numerics.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;netstandard2.1;$(NetFrameworkMinimum)</TargetFrameworks>
```

Comment:
same here: I don't think we need a `NetCoreAppCurrent` TFM.

Reply:
BCL packages Cryptography and TimeProvider have it. From what I remember I think it's so that the forwarding can happen correctly.

Reply:
Ah, I just saw your other comment below. Originally we didn't have the standard2.1 target, that was added after. I'll do what your below suggestion says.

Reply:
You can remove the NetCoreAppCurrent TFM, I'm pretty sure it's not needed. We don't have it in Microsoft.Bcl.AsyncInterfaces either.

Reply:
Removed and updated.